Arrowgene.Ddon.Shared/AssetRepository.cs

```diff
@@ -23,19 +27,37 @@ public AssetRepository(string folder)
             }
 
             ClientErrorCodes = new List<ClientErrorCode>();
-            EnemySpawns = new List<EnemySpawn>();
+            _localEnemySpawns = new List<EnemySpawn>();
+            _spreadsheetEnemySpawns = new List<EnemySpawn>();
+
+            _enemySpawnCsvReader = new EnemySpawnCsvReader();
+            _enemySpawnCsvReader.EnforceCRLF = false;
         }
 
         public List<ClientErrorCode> ClientErrorCodes { get; }
-        public List<EnemySpawn> EnemySpawns { get; }
-
+        public List<EnemySpawn> EnemySpawns
+        {
+            get { return new List<EnemySpawn>(_localEnemySpawns.Concat(_spreadsheetEnemySpawns)); }
+        }
 
         public void Initialize()
         {
             ClientErrorCodes.Clear();
             Load(ClientErrorCodes, "ClientErrorCodes.csv", new ClientErrorCodeCsvReader());
-            Load(EnemySpawns, "EnemySpawn.csv", new EnemySpawnCsvReader());
-            LoadSpreadsheet(EnemySpawns, "1KmwWymqdMGtbRUqu9GvSi_97o-rBj5DJVn2hk7tvs-A", new EnemySpawnCsvReader());
+            Load(_localEnemySpawns, "EnemySpawn.csv", _enemySpawnCsvReader);
+            LoadSpreadsheet(_spreadsheetEnemySpawns, "1KmwWymqdMGtbRUqu9GvSi_97o-rBj5DJVn2hk7tvs-A", _enemySpawnCsvReader);
+
+            // Listen for changes in the enemy csv file and update the spawns
+            FileSystemWatcher watcher = new FileSystemWatcher(_directory.FullName, "EnemySpawn.csv");
```

Comment:
I like this, however I am wondering if this can be more generalized, so we have the `Load` method that is used to populate a list, I am thinking that we could add some code to "attach file watcher to this path if not watched" and then whenever any file changes it will trigger its Load method again.

lastly, the assetRepository could have a event that is exposed, so one can subscribe to the `file reload` event.